packages/hash/api/src/model/ontology/util.ts

```diff
@@ -18,10 +18,10 @@ export const getNamespaceOfAccountOwner = async (
       ? SYSTEM_ACCOUNT_SHORTNAME
```

Comment:
(comment for line 15) Just to check, is this todo still relevant?

Reply:
I don't think it is/made much sense to begin with, as this isn't really a hack. Have removed it in https://github.com/hashintel/hash/pull/1435/commits/ef0e6450135ee8ac8451b6fd76541e2311dd3c3a